### PR TITLE
Added stm32f7

### PR DESCRIPTION
**Pull request by korken89**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -34,6 +35,12 @@ mod pointers {
     pub const FLASH_SIZE_PTR: *const u16 = 0x1FFF_7A22 as _;
 }
 
+#[cfg(feature = "stm32f7")]
```
Comment (Disasm) [inline src/lib.rs]:
```suggestion
#[cfg(any(feature = "stm32f72x", feature = "stm32f73x"))]
```

Comment (korken89) [reply src/lib.rs]: Fixed :)

File: src/lib.rs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 //! * `stm32f2`
 //! * `stm32f3`
 //! * `stm32f4`
+//! * `stm32f7`
```
Comment (Disasm) [inline src/lib.rs]:
```suggestion
//! * `stm32f72x`
//! * `stm32f73x`
```

**Review by Disasm — APPROVED** [review]
Looks good, thank you!

**Pull request merged**